crates/pallet-domains/Cargo.toml

```diff
@@ -28,6 +28,9 @@ subspace-core-primitives = { version = "0.1.0", default-features = false, path =
 
 [dev-dependencies]
 pallet-balances = { version = "4.0.0-dev", git = "https://github.com/subspace/polkadot-sdk", rev = "fb2fd5da02ca6bc86ef7445708c55292084fecd8" }
+pallet-offences-subspace = { version = "0.1.0", path = "../pallet-offences-subspace" }
+pallet-subspace = { version = "0.1.0", path = "../pallet-subspace" }
+pallet-timestamp = { version = "4.0.0-dev", git = "https://github.com/subspace/polkadot-sdk", rev = "fb2fd5da02ca6bc86ef7445708c55292084fecd8" }
```

Comment:
1) On `main` branch `pallet-timestamp` is no longer a dependency of `pallet-subspace` (due to PoT enabled by default)
2) Is `pallet-subspace` dependency really necessary? You should be able to use literally any key for testing purposes